airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/asserts.py

```diff
@@ -23,11 +23,41 @@
 #
 
 import logging
+import re
+import pendulum
 from collections import defaultdict
 from typing import List, Mapping
 
 from airbyte_cdk.models import AirbyteRecordMessage, ConfiguredAirbyteCatalog
-from jsonschema import Draft4Validator, ValidationError
+from jsonschema import Draft7Validator, FormatChecker, ValidationError, FormatError
+
+timestamp_regex = re.compile(("^\d{4}-\d?\d-\d?\d" # date
+                              "(\s|T)" # separator
+                              "\d?\d:\d?\d:\d?\d(.\d+)?" # time
+                              ".*$" #timezone
+                             ))
```

Comment:
is there a standard (e.g. ISO 8601) that this regex is based on? 

Whether based on a standard or custom, it would be useful to add a description in the docs (somewhere [on this page](https://docs.airbyte.io/connector-development) perhaps) so that date/time format requirements of a connector are clear when building one.

Reply:
No, its based on bigquery timestamp format (https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#timestamp_type) as its most strict of sql timestamp formats we are supporting

Reply:
Gotcha, so we're lining it up to our most strict destination to ensure compatibility. Makes sense. Small piece in creating connector docs explaining that (with some examples) would be great.